pkg/handlers/metrics_middleware_test.go

```diff
@@ -0,0 +1,33 @@
+package handlers
+
+import (
+	"net/http"
+	"testing"
+
+	. "github.com/onsi/gomega"
+)
+
+func Test_MetricsMiddleware(t *testing.T) {
+	tests := []struct {
+		name       string
+		wantNotNil bool
+	}{
+		{
+			name:       "Should create metrics middleware handler",
+			wantNotNil: true,
+		},
+	}
+
+	RegisterTestingT(t)
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			req, rw := GetHandlerParams("GET", "/", nil)
+			m := http.NewServeMux()
+			handler := MetricsMiddleware(m)
+			Expect(handler != nil).To(Equal(tt.wantNotNil))
```

Comment:
I find the double negation (handler != nil and wantNotNil) to be confusing.
I believe it would be more readable to have:
`Expect(handler == nil).To(Equal(tt.wantNil)`
or (more verbose)
```go
if tt.wantNil {
    Expect(handler).To(BeNil())
} else {
    Expect(handler).NotTo(BeNil())
}

```